Tests/Functional/Controller/AuthorizeControllerTest.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+namespace Payum\Bundle\PayumBundle\Tests\Functional\Controller;
+
+use Payum\Bundle\PayumBundle\Tests\Functional\WebTestCase;
+use Symfony\Component\HttpKernel\Exception\NotFoundHttpException;
+
+class AuthorizeControllerTest extends WebTestCase
+{
+    /**
+     * @ticket 507
```

Comment:
I don't think this comment provides anything useful.

Reply:
allows to filter tests based on their issue ID in phpunit which is very usefull while writing test and later it provides link to issue where you can often find explanation to why is this test here and what it is supposed to do.

Reply:
Sure, but I think it's only useful in tests covering specific edge cases. Also, only having the ticket id means you have to find the repo and issue number manually, and it might not contain enough information to explain the purpose of the test.

So this annotation can be accompanied by a `@see https://github.com/Payum/PayumBundle/issues/507` to link to the issue directly